.circleci/config.yml

```
@@ -16,6 +16,13 @@ jobs:
             echo 'export PATH=~/.npm-global/bin:$PATH' >> $BASH_ENV
             echo 'export NODE_PATH=~/.npm-global/lib/node_modules:$NODE_PATH' >> $BASH_ENV
 
+      - run:
+          name: "Check git conflicts"
+          command: |
+            git remote add upstream https://github.com/pingcap/docs.git
```

```suggestion
            git remote add upstream https://github.com/pingcap/docs-cn.git
```
Hope you understand the commands here soon.